controllers/managedocs_controller.go

```diff
@@ -1669,6 +1680,16 @@ func (r *ManagedOCSReconciler) findOCSVolumeClaims() (bool, error) {
 	return false, nil
 }
 
+func (r *ManagedOCSReconciler) findStorageConsumerList() (bool, error) {
+	storageConsumerList := ocsv1alpha1.StorageConsumerList{}
+	if err := r.UnrestrictedClient.List(r.ctx, &storageConsumerList); err != nil {
```

Comment:
Why do we need unrestrictedClient to list storageConsumer? All the storageConsumer will be in openshift-storage namespace, right?

Reply:
Yes correct. Made the change